src/helpers/profilePic.js

```diff
@@ -0,0 +1,26 @@
+import multer from 'multer';
+import cloudinary from 'cloudinary';
+import cloudinaryStorage from 'multer-storage-cloudinary';
+import dotenv from 'dotenv';
+import config from '../db/config/config';
+
+const { cloudName, apiKey, apiSecret } = config;
+
+
+dotenv.config();
```

Comment:
There's no need to configure `dotenv` in this file since its been done in the config file where you're importing the items in the environment.